source/manual/architecture-deep-dive.html.md

```diff
@@ -400,33 +400,41 @@ and processes.
 
 Puppet runs in a master/agent setup. There is a single "puppetmaster" running
 on its own class of machine, whereas the agents run on all the other machines
-(irrespective of class). Each puppet agent is responsible for sending alerts
-and logs to our various [monitoring systems][], and for running health check
```

Comment:
Do these reference links actually work with the empty brackets [] ? Usually those are omitted.

Reply:
They work, but it looks like they're optional. I've now removed them 👍